pom.xml

```diff
@@ -505,7 +485,7 @@
             <properties>
                 <skipITs>true</skipITs>
                 <maven.test.skip>true</maven.test.skip>
-                <docker.skip>true</docker.skip>
+                <!-- <docker.skip>false</docker.skip> -->
```

Comment:
For `quick` remove this line. We want `mvn package -Dquick` to create a docker image.